1-js/01-getting-started/2-code-editors/article.md

```diff
@@ -51,12 +51,12 @@ I'm using:
 - [WebStorm](http://www.jetbrains.com/webstorm/) for JS, and if there is one more language in the project, then I switch to other Jetbrains editors like [PHPStorm](http://www.jetbrains.com/phpstorm/) (PHP), [IDEA](http://www.jetbrains.com/idea/) (Java), [RubyMine](http://www.jetbrains.com/ruby/) (Ruby). There are editors for other languages too, but I haven't used them.
 - As a lightweight editor -- [Sublime Text](http://www.sublimetext.com) or [Atom](https://atom.io/).
 
-If you don't know what to choose -- you can consider these ones.
+If you don't know what to choose, you can consider these ones.
 
 ## Let's not argue
 
-The editors in the lists above are those that me or my friends -- good developers have been using for a long time and are happy with.
+The editors in the lists above are those that either my friends or I -- good developers -- have been using for a long time and are happy with.
```

Comment:
That sounds good to me. Just need to add some punctuation. The phrase "who I consider good friends" is an aside, so it should be enclosed in parentheses or dashes. In general, parentheses are used when the aside is incidental, and dashes are used when the aside warrants more attention. The aside does not seem incidental to me, so I would use dashes, but it is a matter of stylistic preference so it is up to you to decide.

In case you are wondering my credentials: I studied rhetoric, communication and English in college, and I am a native English speaker. Thanks for your comments!